libs/stache/src/lib/modules/table-of-contents/table-of-contents.component.spec.ts

```diff
@@ -1,261 +1,223 @@
-import { NO_ERRORS_SCHEMA } from '@angular/core';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
-import { RouterTestingModule } from '@angular/router/testing';
-import { expect } from '@skyux-sdk/testing';
-
-import { of as observableOf } from 'rxjs';
+import { SkyAppTestUtility, expect, expectAsync } from '@skyux-sdk/testing';
```

Comment:
These tests were embarrassingly bad; had to be rewritten due to private property access.